practice-two/src/style/sections/project.css

```diff
@@ -0,0 +1,169 @@
+.project {
+  margin-top: 29px;
+  margin-bottom: 50px;
```

Comment:
Should use padding instead of margin to make consistent with testimonials section